apps/web/src/views/Notifications/containers/OnBoardingView.tsx

```diff
@@ -51,35 +53,52 @@ function OnboardingButton({ onClick, loading, isOnBoarded, account, isReady }: I
   )
 }
 
-const OnBoardingView = ({ identityKey, handleRegistration, isReady }: IOnBoardingProps) => {
+const OnBoardingView = ({
+  identityKey,
+  handleRegistration,
+  isReady,
+  isSubscribing,
+  setIsSubscribing,
+}: IOnBoardingProps) => {
+  const [isOboarding, setIsOnBoarding] = useState<boolean>(false)
   const toast = useToast()
   const { t } = useTranslation()
   const { address: account } = useAccount()
-  const { subscribe, isSubscribing } = useManageSubscription(`eip155:1:${account}`)
-  const { sendPushNotification, requestNotificationPermission, subscribeToPushNotifications } =
-    useSendPushNotification()
+  const { subscribe } = useManageSubscription(`eip155:1:${account}`)
+  const { sendPushNotification, subscribeToPushNotifications } = useSendPushNotification()
 
   const handleSubscribe = useCallback(async () => {
+    setIsSubscribing(true)
     try {
-      await subscribe()
       await subscribeToPushNotifications()
+      await subscribe()
       setTimeout(async () => {
         await sendPushNotification(BuilderNames.OnBoardNotification, [], `eip155:1:${account}`)
-        await requestNotificationPermission()
-      }, 1500)
+      }, 1200)
```

Comment:
what's the reason for the timeout here?

Reply:
need it because the notification doesnt send sometimes immediately after subscribing because the BE will still think user is not subscribed